code/__DEFINES/economy.dm

```diff
@@ -53,7 +53,6 @@
 #define ACCOUNT_ALL_NAME "United Station Budget" // for negative station trait - united budget
 
 // If a vending machine matches its department flag with your bank account's, it gets free.
-#define NO_FREEBIES 0 // used for a vendor selling nothing for free
 #define ACCOUNT_COM_BITFLAG (1<<0) // for Commander only vendor items (i.e. HoP cartridge vendor)
```

Comment:
Are these flags actually used for anything anymore?

Reply:
They can be re-added when they have a use again. 

This fits into the same situation as "every PR should be treated as final and with no follow-up" so you should try to clean things up that are no longer used as of your PR.